app/controllers/meetups_controller.rb

```diff
@@ -48,10 +53,23 @@ def destroy
   private
 
   def meetup_create_params
-    params.require(:meetup).permit(:description, :start_time).merge(
-      court: @court,
-      user: current_user
+    meetup_info = params.permit(
+      :description, :start_day, :start_hour, :start_minute
     )
+
+    date = week_days[meetup_info[:start_day].to_i][0]
+    hour = meetup_info[:start_hour].to_i
+    minute = meetup_info[:start_minute].to_i
+    description = meetup_info[:description]
+
+    datetime = DateTime.new(date.year, date.month, date.day, hour, minute)
```

Comment:
Do not use `DateTime.new` without zone. Use one of `Time.zone.local`, `DateTime.new.current`, `DateTime.new.in_time_zone`, `DateTime.new.utc`, `DateTime.new.getlocal`, `DateTime.new.iso8601`, `DateTime.new.jisx0301`, `DateTime.new.rfc3339`, `DateTime.new.to_i`, `DateTime.new.to_f` instead.